calculate duration, define document structure, send to elasticsearch

nataliemona opened the pull request.

Commit:
calculate duration, define document structure, send to elasticsearch

lbjay commented on meetings.py:
```diff
@@ -16,12 +17,17 @@
 SECRET = getenv('ZOOM_SECRET')
 API_BASE_URL = "https://api.zoom.us/v1"
 MEETING_TYPES = {"live": 1, "past": 2}
+ES_HOST = getenv('ES_HOST')
```
add this to `example.env`

lbjay commented on meetings.py:
```diff
@@ -43,8 +49,15 @@ def get_meetings_from(date, type, key, secret):
+            es.index(
+                index="meetings",
+                doc_type="meeting",
+                id=meeting['uuid'],
```
remind me the diff between `uuid` and `id`? We need to differentiate between the abstract "meeting" and the occurrences of the meetings. I think it's the latter that we want to index.

lbjay commented on meetings.py:
```diff
@@ -60,6 +73,32 @@ def get_meetings_from(date, type, key, secret):
+def create_meeting_document(meeting, type):
+
+    doc = {
+        "reoccurring_id": meeting['id'],
```
`occurrence_id`?

lbjay commented on meetings.py:
```diff
@@ -72,6 +111,19 @@ def open_destination(destination=None):
+
+    return seconds
+
+
```
This is good. For kicks I tried to duplicate using builtin datetime methods.
```
def to_seconds(duration):
    try:
        dt = datetime.strptime(duration, "%H:%M:%S")
    except ValueError:
        dt = datetime.strptime(duration, "%M:%S")
    delta = timedelta(hours=dt.hour, minutes=dt.minute, seconds=dt.second)
    return int(delta.total_seconds())
```

Yours is waay faster but IMO mine takes less cognitive load to read/figure out what it's doing. 

¯\\\_(ツ)\_/¯

Mine:
```
In [30]: %timeit to_seconds("00:32:04")
25.9 µs ± 63.7 ns per loop (mean ± std. dev. of 7 runs, 10000 loops each)

In [31]: %timeit to_seconds("32:04")
38.1 µs ± 104 ns per loop (mean ± std. dev. of 7 runs, 10000 loops each)
```

Yours:
```
In [34]: %timeit to_seconds("00:32:04")
3.06 µs ± 18 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)

In [35]: %timeit to_seconds("00:32:04")
3.06 µs ± 12.6 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)
```

Commit:
Added ES_HOST to example.env

Commit:
Rewrote harvester to capture meeting topic and host ids in meeting documents.
Also now captures sessions (participant sessions) and stores in separate sessions documents.
